package.json

```diff
@@ -86,7 +87,9 @@
     "start-prod": "tt-serve --env.disableHotReloading -p",
     "start-static": "tt-serve-static --site ./build",
     "test": "npm run jest && npm run wdio",
-    "wdio": "NODE_OPTIONS=--max-old-space-size=2048 wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js"
+    "wdio-default": "NODE_OPTIONS=--max-old-space-size=2048 wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js",
+    "wdio-fusion": "NODE_OPTIONS=--max-old-space-size=2048 THEME=orion-fusion-theme wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js",
+    "wdio": "npm run wdio-default && npm run wdio-fusion"
```

Comment:
can we do the same script changes per component? So add `wdio-default` then update `wdio` to run `wdio-default` and `wdio-fusion`.

Reply:
Yeah, thats a good suggestion.
Changes have been done : https://github.com/cerner/terra-core/pull/2702/commits/112e18ff63a3a70ca53dcc1e1840d325d2be83f3